Makefile.toml

```diff
@@ -87,7 +89,10 @@ for path in ${handle}
     root_dir = dirname ${path}
     if not is_empty ${root_dir}
         echo "Automatically generating ${root_dir}/README.md"
-        exec cargo readme -r ${root_dir} -o README.md -t ${template}
+        output = exec cargo readme -r ${root_dir} -o README.md -t ${template}
+        if not equals ${output.code} 0
+            trigger_error "Running cargo readme failed with error: ${output.stderr}"
+        end
```

Comment:
Suggestion: This is fine, but you can also do as it says [in the SDK docs](https://github.com/sagiegurari/duckscript/blob/master/docs/sdk.md#stdprocessexecute):

> If no output variable is set, the command output will be flushed to the main out/err stream of the parent process.
> In addition, in order to fail the command in case of the child process failed, add the --fail-on-error flag.